notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -11,7 +11,6 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```

Comment:
<ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>